### DISCUSSION TITLE
custom extension stopped working

### DISCUSSION THREAD

**Discussion by Sunny-da-moth** [TurboWarp/extensions#581] [Q&A] [answered]
my custom extension stopped working after i tried adding another menu field and it gives me this error in the console
![Screenshot_116](https://github.com/TurboWarp/extensions/assets/130099468/3eaa6fb9-8905-4735-9f93-a47f6c75f616)
i tried commenting the block that uses the menu, the menu, formatting the menu to be a list, renaming the menu, renaming the block, adding comas, removing comas, asking chatgpt but nothing worked
if you would like to download the extension to try it or to fix it then its at [https://drive.google.com/file/d/1AUwD_rrswuuWkQC_WA39NZ13PemEeQr3/view?usp=sharing](url)

**Comment by LilyMakesThings** [comment]
The error is occuring in the block "MathMenuBlock". It's trying to read a menu called "MENUCONSTANTMENU", which does not exist in the extension.

**Reply by Sunny-da-moth** [reply]
now i renamed the menu back, the issue persisted
![Screenshot_118](https://github.com/TurboWarp/extensions/assets/130099468/66b2ba43-3af2-4d7c-a89c-43aa2b29563d)

**Reply by Sunny-da-moth** [reply, marked as answer]
misspelt it again, now it works
